testsuite/documentation/cucumber-steps.md

```diff
@@ -415,6 +415,7 @@ Note that the text area variant handles the new lines characters while the other
   Then service "bind" is enabled on "proxy"
   Then service "dhcpd" is running on "proxy"
   When I restart the "bind" service on "sle_minion"
+  When I reload the "apache2" service on "proxy"
```

Comment:
And

Reply:
I would fix the repeated Then and the repeated When keywords.